src/backend/xrender/xrender.c

```diff
@@ -382,10 +470,11 @@ static void release_image_inner(backend_t *base, struct _xrender_image_data_inne
 	free(inner);
 }
 static void release_image(backend_t *base, void *image) {
-	struct backend_image *img = image;
-	img->inner->refcount--;
-	if (img->inner->refcount == 0) {
-		release_image_inner(base, (void *)img->inner);
+	struct xrender_image *img = image;
+	xcb_free_pixmap(base->c, img->rounded_rectangle);
+	img->rounded_rectangle = XCB_NONE;
+	if (img->base.inner->refcount == 0) {
+		release_image_inner(base, (void *)img->base.inner);
```

Comment:
Is it correct, that we don't reduce the `refcount` of the inner image now? We did so previously.

Reply:
Nice catch!